hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -260,6 +260,19 @@ public void handleRound(
         recordCache.resetRoundReceipts();
         boolean transactionsDispatched = false;
 
+        try {
+            // This is only set if streamMode is BLOCKS or BOTH or once user transactions are handled
+            // Dispatch transplant updates for the nodes in override network
+            if (boundaryStateChangeListener.lastConsensusTime() != null) {
```

Comment:
I think it's more explicit to use `if (streamMode != RECORDS)` and just accept a `RECORDS`-only network will not support override networks after release `0.64`.

Reply:
I used this because we are doing same check in other places to recognize `streamMode != RECORDS`. Would you like me to change it everywhere ?